### ISSUE TITLE
example with multiple traits

### ISSUE THREAD

**Issue by cecileane** [rsetienne/secsse#15]
I tried but could not find an example to use SecSSE with multiple discrete traits --say two traits, each binary. If this is possible, it would be great to show an example in the [vignette](https://cran.r-project.org/web/packages/secsse/vignettes/Using_secsse.html). In particular, it would be great to show an example to setup the constraints on which parameters are set to be equal or 0 etc.

My apologies if I misunderstood that this was an option (other than manually re-coding the 2 traits as a single composite trait...?) Thanks!

**Comment by rsetienne** [comment]
There is no function that does this because there are so many ways in which
one can have multiple traits and how speciation and extinction rates depend
on them.
But we'll think about the vignette suggestions.


Op ma 15 nov. 2021 00:53 schreef Cécile Ané ***@***.***>:

**Comment by cecileane** [comment]
Yes, I can see how possibilities are endless, and how they can all be achieved by constructing a new trait with all the original trait combinations, and using your flexible setup to build parameter constraints. Thank you @leonelhalsina for your offline suggestions!
